package.json

```diff
@@ -36,91 +36,73 @@
     "migration:show": "npm run typeorm migration:show"
   },
   "dependencies": {
-    "@jolocom/sdk": "^1.2.0",
```

Comment:
Please leave the jolocom dependencies in

Reply:
I don't think we should until they fix their issues and update their packages. They haven't updated multiple packages in ages (e.g. typeorm dependency 0.2.24 is causing issues with our upgrade, and current typeorm version is 0.2.35, which is conflicting to theirs). Running dependency analyzer shows we are not actively using it either so it's just a nuisance - I suggest we re-add it once it is ready to be used.

Reply:
ok, lets not hold up this PR then - can you please raise another issue with the problems you ran into when using their components with latest version of typeorm?